.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "inotify"]
 	path = inotify
-	url = git@github.com:dsoprea/PyInotify.git
+	url = https://github.com/dsoprea/PyInotify.git
```

Comment:
hmm, this is a problem. You do not want to depend on a local relative git repo somewhere. Maybe we can make this conditional?

Reply:
lets discuss this at some point outside of this ticket, it was just a reminder for us :)
And I agree, I don't think relative URL's work for local cloning as it will interpret it as file://../..